bin/update_maistra_deps.sh

```diff
@@ -23,25 +23,16 @@ go get -d "maistra.io/api@${UPDATE_BRANCH}"
 go mod tidy
 go mod vendor
 
-# Copy CRD's
-dir=$(mktemp -d)
-git clone "https://github.com/maistra/api.git" "${dir}"
-pushd "${dir}"
-git checkout "${UPDATE_BRANCH}"
-popd
-
 # FIXME: https://issues.redhat.com/browse/MAISTRA-2353
```

Comment:
What about this issue? Are we going to copy all of maistra CRDs?

Reply:
no - as I commented on the issue, I don't think it makes sense to do that. we should discuss in the team what we want to do and then remove the FIXME when we address the issue.